docs/dev-guide/3.0-development/data-modeling.rst

```diff
@@ -0,0 +1,8 @@
+.. warning::
+    All documents within the 3.0-Development section should be considered temporary.
+
+
+Data Modeling
+===============
```

Comment:
I'm thinking the content of  `models/README.md `could be moved here.  The `Introduction` and `Development Process` sections could be included in this document.  Not sure what to do with `Notes About Models` section.  We could include them in the document as well.  Or as separate documents that are included in the index.  Separate documents might reduce the complexity of merge conflicts.  Thoughts?